### PR TITLE
Revert catto's change for now

### PR DESCRIPTION
**Pull request by minz1027**
- Revert @catto 's [change](https://github.com/screwdriver-cd/data-schema/commit/1d72e57df84311358452f34a83bc043db4981057) for now
- We are not ready for it yet

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-data-schema",
-  "version": "13.0.0",
+  "version": "12.3.0",
```
Comment (FenrirUnbound) [inline package.json]: Well this is going to be awkward. That means that we have to keep track of the fact that version 13 exists, while the `master` branch shows version 12 is the latest.

Comment (minz1027) [inline package.json]: Keep version 13.0.0 but revert catto's change. Is that alright?

Comment (FenrirUnbound) [inline package.json]: catto's update was valid. why undo it?

Comment (FenrirUnbound) [inline package.json]: Why can't we patch 12.x with what we need?

Comment (stjohnjohnson) [inline package.json]: I think because it's still WIP, we're in a inconsistent state.  I'd revert the change, keep at 13, and open a PR to revert the revert when ready.
